src/validations/rules/sar.sch

```diff
@@ -296,6 +301,7 @@
             <sch:assert
                 diagnostics="has-acceptable-security-assessment-plan-rlink-media-type-diagnostic"
                 doc:guide-reference="Guide to OSCAL-based FedRAMP Security Assessment Results (SAR) §3.5"
+                fedramp:specific="true"
```

Comment:
This is specific to OSCAL (not FedRAMP). All of the inter-document elements are fundamental OSCAL constructs.

Reply:
OSCAL media-types are only constrained by the Media Type Registry.  But the FedRAMP Guide states only those two are acceptable for this resource context.

Reply:
Agreed.